### PR TITLE
Remove /var/lib/wicked/*.xml also on svirt backend

### PR DESCRIPTION
**Pull request by pdostal**
- Related ticket: [poo#116770](https://progress.opensuse.org/issues/116770)
- Verification run: [create_hdd_textmode](https://openqa.suse.de/tests/9716078), [extra_tests_textmode_mod_desktop](https://openqa.suse.de/tests/9715835), [extra_tests_textmode](https://openqa.suse.de/tests/9716177), [yast2_ncurses_textmode](https://openqa.suse.de/tests/9716178)

### CODE REVIEW

File: tests/shutdown/cleanup_before_shutdown.pm [hunk 1]
```diff
@@ -50,16 +50,16 @@ END_SCRIPT
 
     # Proceed with dhcp cleanup on qemu backend only.
     # Cleanup is made, because if same hdd image used in multimachine scenario
-    # on several nodes, the dhcp clients use same id and cause conflicts on dhcpd server.
-    if (is_qemu) {
+    # on several nodes, the dhcp clients use same DUID and cause conflicts on dhcpd server.
+    if (is_qemu || is_svirt) {
```
Comment (rfan1) [inline tests/shutdown/cleanup_before_shutdown.pm:54]: If we don't need this change on s390x, how about `is_svirt_except_s390x`

**Comment by rfan1** [comment]
I suggest adding some comments in line 12.

File: tests/shutdown/cleanup_before_shutdown.pm (hunk 1 above)
Comment (pdostal) [reply tests/shutdown/cleanup_before_shutdown.pm:54]: Does it make any difference on s390? Can you give me example test run I can verify on?

File: tests/shutdown/cleanup_before_shutdown.pm [hunk 2]
```diff
@@ -50,16 +50,16 @@ END_SCRIPT
...
         my $network_status = script_output('systemctl status network');
         # Do dhcp cleanup for wicked
         if ($network_status =~ /wicked/) {
             systemctl 'stop network.service';
             systemctl 'stop wickedd.service';
             assert_script_run('ls /var/lib/wicked/');
             save_screenshot;
-            script_run('rm -f /var/lib/wicked/*.xml');
+            script_run('rm -f /var/lib/wicked/duid.xml');
```
[the remark this answers is not in the document]
Comment (pdostal) [reply tests/shutdown/cleanup_before_shutdown.pm]: This is probably good point. Let's try to remove also `lease-*.xml` - verification run: https://openqa.suse.de/tests/9715937

File: tests/shutdown/cleanup_before_shutdown.pm (hunk 1 above)
Comment (rfan1) [reply tests/shutdown/cleanup_before_shutdown.pm:54]: for 390x, we use static ip address  defined in https://gitlab.suse.de/openqa/salt-pillars-openqa/-/blob/master/openqa/workerconf.sls

File: tests/shutdown/cleanup_before_shutdown.pm (hunk 2 above)
[the remark this answers is not in the document]
Comment (rfan1) [reply tests/shutdown/cleanup_before_shutdown.pm]: Cool, one more comment, to make sure the code can work perfectly, can you please start 1 -2 more jobs which use the same qcow2 image in parallel?

File: tests/shutdown/cleanup_before_shutdown.pm (hunk 1 above)
Comment (rfan1) [reply tests/shutdown/cleanup_before_shutdown.pm:55]: Anyway, I don't think use `is_svirt` can lead to any issue, we can use this workaround for s390x as well.

Comment (pdostal) [reply tests/shutdown/cleanup_before_shutdown.pm:55]: That's why we do `rm -f` so if those files are missing we will just proceed. Can you give me example test run I can verify on?

File: tests/shutdown/cleanup_before_shutdown.pm (hunk 2 above)
[the remark this answers is not in the document]
Comment (pdostal) [reply tests/shutdown/cleanup_before_shutdown.pm]: Good idea. I will.

File: tests/shutdown/cleanup_before_shutdown.pm (hunk 1 above)
Comment (rfan1) [reply tests/shutdown/cleanup_before_shutdown.pm:55]: https://openqa.suse.de/tests/9678944#dependencies

**Pull request merged**